homeassistant/components/ecobee/climate.py

```diff
@@ -271,6 +275,12 @@ def resume_program_set_service(service):
         "set_dst_mode",
     )
 
+    platform.async_register_entity_service(
```

Comment:
The humidifier features should be exposed as a humidifier entity instead.

https://developers.home-assistant.io/docs/core/entity/humidifier

Reply:
Let me know if this looks OK.